### ISSUE TITLE
Determine if "kind" can be "ManageIQ" as opposed to "Manageiq"

### ISSUE THREAD

**Issue by carbonin** [ManageIQ/manageiq-pods#406]
https://github.com/ManageIQ/manageiq-pods/pull/389#discussion_r397324661

**Comment by gyliu513** [comment]
@carbonin @Fryguy I did some test, yes, we can use `ManageIQ`, and I think this `kind` name is better.

```console
root@gyliu-c11:~/go/src/github.com/gyliu513/manageiq-operator# operator-sdk add api --api-version=manageiq.org/v1alpha1 --kind=ManageIQ
INFO[0000] Generating api version manageiq.org/v1alpha1 for kind ManageIQ.
INFO[0000] Created pkg/apis/manageiq/group.go
INFO[0000] Created pkg/apis/manageiq/v1alpha1/manageiq_types.go
INFO[0000] Created pkg/apis/addtoscheme_manageiq_v1alpha1.go
INFO[0000] Created pkg/apis/manageiq/v1alpha1/register.go
INFO[0000] Created pkg/apis/manageiq/v1alpha1/doc.go
INFO[0000] Created deploy/crds/manageiq.org_v1alpha1_manageiq_cr.yaml
INFO[0000] Running deepcopy code-generation for Custom Resource group versions: [manageiq:[v1alpha1], ]
INFO[0009] Code-generation complete.
INFO[0009] Running CRD generator.
INFO[0009] CRD generation complete.
INFO[0009] API generation complete.
```

**Comment by carbonin** [comment]
@gyliu513 is this addressed in https://github.com/ManageIQ/manageiq-pods/pull/432 ?

**Comment by gyliu513** [comment]
@carbonin  No, #442 was still using Manageiq now, perhaps I can create another PR for this after #432 got merged?
